tf_encrypted/keras/models/sequential.py

```diff
@@ -101,3 +103,66 @@ def layers(self):
     if layers and isinstance(layers[0], InputLayer):
       return layers[1:]
     return layers[:]
+
+  def set_weights(self, keras_weights, sess):
+    """ Sets the weights of the model.
+    Arguments:
+      weights: A list of Numpy arrays with shapes and types
+          matching the output of model.get_weights()
+      sess: tfe session"""
+
+    # List model weights
+    model_weights = []
+    for l in self.layers:
+      model_weights += l.weights
+
+    # Define keras weights as private variables
+    keras_weights_p = [tf_encrypted.define_private_variable(w)
+                       for w in keras_weights]
+
+    # Assign new keras weights to existing weights defined by
+    # default when tfe layer instantiated
+    sess.run(tf.global_variables_initializer())
+    for i, w in enumerate(model_weights):
+      sess.run(tf_encrypted.assign(w, keras_weights_p[i]))
+
+  @staticmethod
+  def from_config(keras_config):
```

Comment:
currently to transform the keras model into tfe.keras model we have to do:
`tfe_model = tfe.keras.Sequential.from_config(config)`

Do we want also have the option to do:
- `tfe_model = tfe.keras.models.from_config(config)`

If yes, is it possible to add a static method to a module?

Reply:
I think we should yes! as well as [`clone_model`](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/models/clone_model)

Reply:
to add (static) methods to a module you simply define the function in a file:

```python
# in file tf_encrypted/keras/models.py
def clone_model(model, input_tensors=None):
  ...
```

Reply:
@mortendahl, when you say "yes" you mean `from config` should be a classmethod?

Reply:
TF seems to have two calls for this, suspect one calls the other underneat:
- https://www.tensorflow.org/api_docs/python/tf/keras/models/model_from_config
- https://www.tensorflow.org/api_docs/python/tf/keras/models/Model#from_config

Reply:
ah great, thanks for sharing this.